Automattic/pocket-casts-android
# Fix missing margin on search screens

## Description

When the mini player is open, the search screens display the bottom of the page under the mini player. This is happening with the search page, the podcast results page, and the episode results page.

Fixes https://github.com/Automattic/pocket-casts-android/issues/2648

## Testing Instructions

### Test with mini player
1. Play an episode so the mini player is open
2. Tap the Podcasts tab
3. Tap the search icon
4. Enter a search term such as 'test'
5. Scroll to the bottom of the page

✅ Verify the last episode in the search results appears

6. Scroll to the top
7. Tap the "Show all" link next to the podcasts
8. Scroll to the bottom of the page

✅ Verify the last podcast in the search results appears

9. Navigate back
10. Tap the "Show all" link next to the episodes
11. Scroll to the bottom of the page

✅ Verify the last episode in the search results appears

Repeat the steps with the mini player closed.

## Screenshots

| Before | After |
| --- | --- |
| ![Screenshot_20240816_111945](https://github.com/user-attachments/assets/946a6ab4-751e-4b4c-bd35-57e700ab4751) | ![Screenshot_20240816_111836](https://github.com/user-attachments/assets/11b3fef7-a708-4f14-b6db-978662ff1f1b) |
